draft-ietf-sidrops-bar-sav.xml

```diff
@@ -20,17 +20,29 @@
 <!-- If further character entities are required then they should be added to the DOCTYPE above.
      Use of an external entity file is not recommended. -->
 
+<!--
 <rfc
   xmlns:xi="http://www.w3.org/2001/XInclude"
   category="bcp"
-  docName="draft-ietf-sidrops-bar-sav-latest"
+  docName="draft-ietf-sidrops-bar-sav-06"
```

Comment:
This must remain "latest", while the actual version is indicated by a tag.
I'll fix this.